sbt-scalajs-bundler/src/main/scala/scalajsbundler/Stats.scala

```diff
@@ -50,18 +49,18 @@ object Stats {
 
   }
 
-  final case class WebpackError(moduleName: String, message: String, loc: String)
+  final case class WebpackError(moduleName: Option[String], message: String, loc: String)
```

Comment:
the docs are a little unclear, but besides `message` it sounds like "various" other properties means perhaps `loc` should be optional too?:
https://webpack.js.org/api/stats/#errors-and-warnings